vertica.py

```diff
@@ -0,0 +1,28 @@
+# pip install vertica-python
+
+from vertica_python import connect
+import json
+connection = connect({
+    'host': '.mozilla.com',
+    'port': 5433,
+    'user': '',
+    'password': '',
+    'database': ''
+
+})
+
+
+def writePartialJSON(channel, exportFile):
+    query = "SELECT version, SUM(tTotalProfiles) AS ADI FROM fhr_rollups_daily_base WHERE stdchannel='%s' GROUP BY timeEnd,version ORDER BY timeEnd desc, SUM(tTotalProfiles) DESC LIMIT 5" % (channel)
+    cur = connection.cursor("dict")
+    cur.execute(query)
+
+    lastReleases = cur.fetchall()
+
+    with open(exportFile, 'w') as outfile:
+        json.dump(lastReleases, outfile)
+
+writePartialJSON("esr", "partial-esr.json")
+writePartialJSON("release", "partial-release.json")
+writePartialJSON("beta", "partial-beta.json")
+
```

Comment:
Is there a reason to use different files per channel? It's probably going to be simpler to deploy if we just have one. It's not the end of the world though.